core/src/commonMain/kotlin/com/sourcepoint/mobile_core/Coordinator.kt

```diff
@@ -393,6 +427,13 @@ class Coordinator(
                                 hasLocalData = state.hasUSNatLocalData,
                                 consentStatus = state.usNat.consents.consentStatus
                             )
+                        },
+                        preferences = campaigns.preferences?.let { 
+                            MessagesRequest.Body.Campaigns.Preferences(
+                                targetingParams = emptyMap(),
+                                hasLocalData = false,
+                                consentStatus = ConsentStatus()
```

Comment:
Is this `ConsentStatus` the same class as for the other legislations? I have the feeling Preferences should have a difference class for status, no?

Reply:
`ConsentStatus` is sent as an empty object because, without it, BE gives an error: 
>"Error while returning Preferences campaign data: TypeError: Cannot read properties of undefined (reading 'replace')"

Reply:
What about using a `emptyMap()` instead? I'm just concerned if we change `ConsentStatus` in the future, it might have side-effects on the call to preferences. We can also ask Dan about why `consentStatus` object is required in that call.

Reply:
@andresilveirah done!